app/src/test/java/org/simple/clinic/summary/PatientSummaryEffectHandlerTest.kt

```diff
@@ -327,18 +328,28 @@ class PatientSummaryEffectHandlerTest {
     )
     val prescriptions = listOf(prescription)
 
+    val medicalHistoryUuid = UUID.fromString("aad78bf5-d96d-4d52-9efd-43a1b59ea139")
+    val medicalHistory = TestData.medicalHistory(
+        uuid = medicalHistoryUuid,
+        patientUuid = patientUuid,
+        diagnosedWithHypertension = Answer.Yes,
+        hasDiabetes = Answer.No
+    )
+
     val patientInformation = PatientTeleconsultationInfo(
         patientUuid = patientUuid,
         bpPassport = bpPassport.identifier.displayValue(),
         facility = facility,
         bloodPressures = bloodPressures,
         bloodSugars = bloodSugars,
-        prescriptions = prescriptions
+        prescriptions = prescriptions,
+        medicalHistory = medicalHistory
     )
 
     whenever(bloodPressureRepository.newestMeasurementsForPatientImmediate(patientUuid, patientSummaryConfig.numberOfMeasurementsForTeleconsultation)) doReturn bloodPressures
     whenever(prescriptionRepository.newestPrescriptionsForPatientImmediate(patientUuid)) doReturn prescriptions
     whenever(bloodSugarRepository.latestMeasurementsImmediate(patientUuid, patientSummaryConfig.numberOfMeasurementsForTeleconsultation)) doReturn bloodSugars
+    whenever(medicalHistoryRepository.historyForPatientOrDefaultImmediate(uuidGenerator.v4(), patientUuid)) doReturn medicalHistory
```

Comment:
The first parameter here should be the `medicalHistoryUuid`, right? It is an implementation detail that it will be the same in the test, but the method stub should be explicit.

Reply:
Ah right, forgot about that. 😅